react/react1/week2/react1-week2/src/TodoListStyle.css

```diff
@@ -0,0 +1,72 @@
+@import url("https://fonts.googleapis.com/css2?family=Pacifico&display=swap");
```

Comment:
if you used App as your main page you could cut this code and paste it on App.css